js/utilities.js

```diff
@@ -1272,13 +1274,13 @@ var ERMrest = (function(module) {
             });
         }
 
-        var conditionalRegex = /\{\{(#|\^)([\w\d-]+)\}\}/;
+        var conditionalRegex = /\{\{(#|\^)([\w\d-_. ]+)\}\}/;
 
         // If no conditional Mustache statements of the form {{#var}}{{/var}} or {{^var}}{{/var}} not found then do direct null check
-        if (!conditionalRegex.exec(template)) {
+        if (!options.avoidValidation && !conditionalRegex.exec(template)) {
```

Comment:
Yes. I added it so that we don't validate the template_url for file upload again which would simply return null.

Fileupload has a method to validate the url which is called by Chaise before it begins upload process.